go/tools/builders/read.go

```diff
@@ -198,6 +198,28 @@ func (r *importReader) findEmbed(first bool) bool {
 		case ' ', '\t':
 			// leave startLine alone
 
+		case '\'':
+			startLine = false
+			for r.err == nil {
+				if r.eof {
+					r.syntaxError()
+				}
+				c = r.readByteNoBuf()
+				if c == '\\' {
+					r.readByteNoBuf()
```

Comment:
```suggestion
					_ = r.readByteNoBuf()
```

Let's show that we're suppressing